### PR TITLE
docs: add links

### PR DESCRIPTION
**Pull request by gao-sun**
add the missing links for integration and connectors

image

### CODE REVIEW

File: docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx [hunk 1]
```diff
@@ -34,9 +34,25 @@ If you skip the guide in Admin Console by accident, you can also pick the guide
...
+      title: 'Single Page App',
+      items: [
+        { key: 'Vanilla JS', link: '/docs/recipes/integrate-logto/vanilla-js' },
+        { key: 'React', link: '/docs/recipes/integrate-logto/react' },
+        { key: 'Vue', link: '/docs/recipes/integrate-logto/vue' },
+      ],
+    },
+    {
+      title: 'Tranditional Web App',
+      items: [{ key: 'Traditional Wen', link: '/docs/recipes/integrate-logto/traditional' }],
```
Comment (darcyYe) [inline docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx:54]:
```suggestion
      items: [{ key: 'Traditional Web', link: '/docs/recipes/integrate-logto/traditional' }],
```

File: docs/docs/tutorials/get-started/enable-passcode-sign-in.mdx [hunk 2]
```diff
@@ -23,10 +23,35 @@ Logto has some built-in connectors which allows out-of-box usage:
...
+          link: 'https://github.com/logto-io/logto/tree/master/packages/connector-aliyun-sms',
+        },
+      ],
+    },
     {
       title: 'Email connectors',
-      items: [{ key: 'Sendgrid Mail' }, { key: 'Aliyun DM' }, { key: 'SMTP' }],
+      items: [
+        {
+          key: 'Sendgrid Mail',
```
Comment (darcyYe) [inline docs/docs/tutorials/get-started/enable-passcode-sign-in.mdx:43]: nit: maybe `SendGrid`
```suggestion
          key: 'SendGrid Mail',
```

**Review by wangsijie — APPROVED** [review]
LGTM

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-passcode-sign-in.mdx [hunk 3]
```diff
@@ -21,10 +21,35 @@ import ConnectorsWip from './fragments/_connectors-wip.md';
...
+          link: 'https://github.com/logto-io/logto/tree/master/packages/connector-aliyun-sms',
+        },
+      ],
+    },
     {
       title: '邮件连接器',
-      items: [{ key: 'Sendgrid 邮件服务' }, { key: '阿里云 DM' }, { key: 'SMTP' }],
+      items: [
+        {
+          key: 'Sendgrid 邮件服务',
```
Comment (darcyYe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-passcode-sign-in.mdx:41]: nit: maybe align with `SendGrid`
```suggestion
          key: 'SendGrid 邮件服务',
```

**Pull request merged**